.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,25 @@
+name: Release Drafter
+
+on:
+  push:
```

Comment:
What happens if someone force pushes to the master branch or reverts a PR? Would it mess up the draft release?

Reply:
I think it won't mess up anything because it always regenerates the release note based on the current state, so an updated master will mean that the release note will be generated based on the updated state

Reply:
Got it, thanks!